app/assets/stylesheets/application.scss

```diff
@@ -4,5 +4,7 @@
  * the top of the compiled file, but it's generally better to create a new file per style scope.
  *= require jquery-ui
  *= require_self
- *= require_tree . 
-*/
\ No newline at end of file
+*/
+@import "dialog-polyfill.css";
+@import "material.min.css";
```

Comment:
I'd leave it up to you guys :) I just thought I'd use Material Design. If you guys prefer Bootstrap I'm fine with it too.